themes/gohugoioTheme/layouts/_default/baseof.html

```diff
@@ -33,10 +33,6 @@
       {{- template "_internal/schema.html" . -}}
       {{- template "_internal/twitter_cards.html" . -}}
 
-      {{ if eq (getenv "HUGO_ENV") "production" | or (eq .Site.Params.env "production")  }}
```

Comment:
We should not modify the base theme. Extend the theme instead - the way we have everything else. Otherwise, it's harder to maintain.